okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -972,6 +979,35 @@ static HttpUrl getChecked(String url) throws MalformedURLException, UnknownHostE
     return PublicSuffixDatabase.get().getEffectiveTldPlusOne(host);
   }
 
+  public static String displayHost(Builder.IDNMode idnMode, String host) {
+    switch (idnMode) {
+      case SAFE:
```

Comment:
supernit: use `int` constants? Since this isn’t public API it’s an easy way to shave a few bytes off the binary